.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Can you add `.idea` to the `.gitignore` in the root folder and remove these files? Should be only local to your working copye.

Reply:
done